### DISCUSSION TITLE
Status of Project

### DISCUSSION THREAD

**Discussion by bryanguillen** [atomiks/tippyjs-react#370] [Q&A]
Hello,

Is this project still being maintained?

There are a few reasons for why I ask:

1. I see a few issues open that have been open for a few months
2. The last release was a year ago
3. The `README` recommends we use Floating UI

It feels like the answer is yes, but just wanted to double check, in case I was missing some context of any sort.  My main concern is that this will soon be "_not maintained", because of Floating UI.

Anyways, I absolutely love this package and have used it in the past.  My use case is super simple, just simple tooltips on hover within a few views, but this package is super helpful b/c those few views are pretty tricky.  But yeah, before I commit to using this within my code base, just wanted to check in on the status of the project.

Thanks,

Bryan G

**Comment by atomiks** [comment]
Hey yeah it's not maintained anymore sorry

[Floating UI has a tooltip guide](https://floating-ui.com/docs/tooltip) which doesn't take much to set up — depends what features of Tippy you were using though.

**Reply by bryanguillen** [reply]
@atomiks , ah got it.  Yeah that makes sense, since popper isn't either.  Thanks for the heads up!
